packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/signingout.md

```diff
@@ -0,0 +1,95 @@
+This singleton stores the token in User Defaults. For more security, store the token in the Keychain.
```

Comment:
is it User Defaults or user defaults? I see it both ways in the doc.

Reply:
Thanks for catching, I thought it was used as a proper noun but a quick look at Apple docs says it's not. Fixed in the next build.